src/transformState.js

```diff
@@ -6,6 +6,25 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const obj of actions) {
+    if (obj['type'] === 'addProperties') {
```

Comment:
it is better to use switch/case if you have limited amount of conditions.
It was on the checklist too